stable/sentry/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Sentry is a cross-platform crash reporting and aggregation platform.
 name: sentry
-version: 0.4.1
+version: 0.4.2
```

Comment:
it is a new functionality please bump to 0.5.0

Reply:
done!